packages/pieces/http/src/lib/actions/return-response.ts

```diff
@@ -0,0 +1,33 @@
+import { Property, createAction } from '@activepieces/pieces-framework';
+import { StatusCodes } from 'http-status-codes';
+
+export const httpReturnResponse = createAction({
```

Comment:
Does it really make sense that this is an action of the http client? Shouldn't there rather be a webhook piece for this?

Reply:
We thought since people are http request, they would expect that return is in http piece.

We also modified the description so it's searchable, I think we can go with that and see if people finds it